server/utils/db/connection_pool.py

```
@@ -85,17 +91,25 @@ def get_connection(self):
             if connection:
                 try:
                     connection.rollback()
-                except:
+                except Exception:
                     pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)